### ISSUE TITLE
events bug in v65

### ISSUE THREAD

**Issue by Owyn** [neowutran/ShinraMeter#272]
On 3rd phase of 30 raid for both black and red buffs it says TTS for red buff only (it says "Красный дебафф" for both colors of buff), but buff IDs seem to be correct in details

```
<!-- Raid 30, phase 3, black debuff (Dust) -->
	<abnormality ingame="true" trigger="Added" target="Self">
		<abnormalities>
			<abnormality>950069</abnormality>
			<abnormality>950180</abnormality>
		</abnormalities>
		<actions>
			<notify>
				<balloon title_text="{abnormality_name}" body_text="{abnormality_name}" display_time="3000" />
				<text_to_speech text="Черный дебафф" />
			</notify>
		</actions>
	</abnormality>
	
	<!-- Raid 30, phase 3, red debuff (Flame) -->
	<abnormality ingame="true" trigger="Added" target="Self">
		<abnormalities>
			<abnormality>950068</abnormality>
			<abnormality>950165</abnormality>
		</abnormalities>
		<actions>
			<notify>
				<balloon title_text="{abnormality_name}" body_text="{abnormality_name}" display_time="3000" />
				<text_to_speech text="Красный дебафф" />
			</notify>
		</actions>
	</abnormality>
```
![41 _proc](https://cloud.githubusercontent.com/assets/1309656/21154617/12285cda-c180-11e6-82da-43c2d769b0f7.jpg)
![40 _proc](https://cloud.githubusercontent.com/assets/1309656/21154619/122ae022-c180-11e6-8bb1-4842460702f5.jpg)
![34 _proc](https://cloud.githubusercontent.com/assets/1309656/21154692/5af43830-c180-11e6-965e-7761f59d0a52.jpg)
![33 _proc](https://cloud.githubusercontent.com/assets/1309656/21154621/122eba12-c180-11e6-9846-4a9404fbaf36.jpg)
![28 _proc](https://cloud.githubusercontent.com/assets/1309656/21154620/122b098a-c180-11e6-9dd0-3d0331c648f9.jpg)
![31 _proc](https://cloud.githubusercontent.com/assets/1309656/21154618/1229eb36-c180-11e6-8740-ca16d68a5061.jpg)

**Comment by neowutran** [comment]
try
```xml
	<!-- Raid 30, phase 3 & 4, black debuff (Dust) -->
	<abnormality  priority="10" ingame="true" active="true" trigger="Added" target="Self">
		<abnormalities>
			<abnormality>950069</abnormality>
			<abnormality>950180</abnormality>
		</abnormalities>
		<actions>
			<notify>
				<balloon title_text="{abnormality_name}" body_text="{abnormality_name}" display_time="3000" />
				<text_to_speech text="Black" />
			</notify>
		</actions>
	</abnormality>
	
	<!-- Raid 30, phase 3 & 4, red debuff (Flame) -->
	<abnormality ingame="true" priority="10" trigger="Added" target="Self">
		<abnormalities>
			<abnormality>950068</abnormality>
			<abnormality>950179</abnormality>
		</abnormalities>
		<actions>
			<notify>
				<balloon title_text="{abnormality_name}" body_text="{abnormality_name}" display_time="3000" />
				<text_to_speech text="Red" />
			</notify>
		</actions>
	</abnormality>
```

anything related to raid 30 is a pain in the ass to test. Only few minutes every weeks x)

**Comment by Owyn** [comment]
Just wanted to add that in previous version of Shinra with the same events file (I haven't changed it) it announced two debuffs right

**Comment by neowutran** [comment]
Just checked in raid 30 -> I confirm issue with the event system for the color. Searching where is the problem

If you can, try this config with the 1.65

```xml
<!-- Raid 30, phase 3 & 4, black debuff (Dust) -->
	<abnormality  priority="10" ingame="true" active="true" trigger="Added" target="Self">
		<abnormalities>
			<abnormality>950069</abnormality>

		</abnormalities>
		<actions>
			<notify>
				<balloon title_text="{abnormality_name}" body_text="{abnormality_name}" display_time="3000" />
				<text_to_speech text="Black" />
			</notify>
		</actions>
	</abnormality>
	
	<!-- Raid 30, phase 3 & 4, red debuff (Flame) -->
	<abnormality ingame="true" priority="10" trigger="Added" target="Self">
		<abnormalities>
			<abnormality>950068</abnormality>
		</abnormalities>
		<actions>
			<notify>
				<balloon title_text="{abnormality_name}" body_text="{abnormality_name}" display_time="3000" />
				<text_to_speech text="Red" />
			</notify>
		</actions>
	</abnormality>

```

**Comment by Gl0** [comment]
Should be fixed in 1.66
